blind_walking/net/feature_encoder.py

```diff
@@ -6,6 +6,28 @@
 from stable_baselines3.common.torch_layers import BaseFeaturesExtractor
 from stable_baselines3.common.torch_layers import create_mlp
 
```

Comment:
trivial, but convention is to have two blank lines between the import statements and other code

Reply:
sure, implemented